### DISCUSSION TITLE
help with committing using git bash on windows 11

### DISCUSSION THREAD

**Discussion by ScareKrwoe** [community/community#136742] [Programming Help] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I'm trying to get git bash to work so I can follow a tutorial on learning version control, but I cannot figure out why I'm encountering these issues. it's been like 15 days I'm losing my mind!

issue #1 After a fresh install of git bash, I select open git bash and the readme at the end of the application window. A window opens and closes instantly.  opening git bash from the start menu results in the same crash every time without fail.  The only way for me to get an admin level git bash open is through typing bash in an elevated command prompt or navigating to bash directory and creating an admin level shortcut. Also, I can right click open bash here, but I think it's not admin level.  which leads me to my second issue.

issue #2 git bash does not save my global config setting, though cmd I was able to edit my username and email once with the git config --global user.name "myname" and the git config --global user.email "myemail" commands. this worked for the duration of the session and then never worked again. now when I right click open bash here or use my admin shortcut, I can't even edit the global configuration.
i get an error  
![git error](https://github.com/user-attachments/assets/9697d1f0-e7bf-4255-a7a4-3c6261ef03fe)

I can't continue trying to learn this wonderful software until I can figure out what the heck is going on. Every video I see nobody has a problem doing this and they achieve this in 15 seconds. I was trying to follow the tutorial to commit but it didn't work so I found that git wanted me to identify myself, but I can't.  I would appreciate some help and I apologize if this is the wrong place to ask questions. it seemed like a good place to start.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by davevad93** [comment]
Hi @ScareKrwoe ,

Did you run `Git Bash` as administrator? If not, do it and see if it fixes the issue.

EDIT: I'm sorry I misread and apparently you have indeed run it as admin.

**Reply by davevad93** [reply, marked as answer]
Go at the root of your C: user folder and check if the `gitconfig` file exists, also check this StackOverflow post which deals with a quite similar issue and you may find it useful:

- [Trouble setting up git with my GitHub Account error: could not lock config file](https://stackoverflow.com/questions/14959972/trouble-setting-up-git-with-my-github-account-error-could-not-lock-config-file)

**Reply by ScareKrwoe** [reply]
thank you for the replay, yes, I have tried so many things, i even fresh installed windows, I'm thinking it's a problem where something is not configured on my system properly. I'm just a complete git noob and i don't have much command line experience either. my research on this problem has led me to think it's an issue with wsl but i don't know enough about the topic to even tell really. the only thing I know is that the version of git on wsl terminal is different than my git bash terminal version which leads me to think they are separate, plush the commands I was using for git bash on windows did not work in wsl application.  do you think trying an older version may help?

**Reply by ScareKrwoe** [reply]
we sent those at the same time so ima read through that and ill get back to you afterwards. i did however confirm that the file was there before and also tried to manually edit in the credentials with notepad++ git bash did not recognize the changes before or after a restart.

**Reply by davevad93** [reply]
You're welcome @ScareKrwoe you may not have permission to the `$HOME` path on your system, in the StackOverflow post I provided above there are a couple of examples of that, if none of the solutions found there work, try removing `git` completely from your system and reinstall it from scratch.

**Reply by ScareKrwoe** [reply]
This helped. Im overjoyed thank you so much it was an issue with the home path,  its all working good now.
![solution](https://github.com/user-attachments/assets/b36ed780-3bf7-4628-88bf-78d672c50c70)

this part right here got it going perfectly. you sir are a life saver!

**Reply by davevad93** [reply]
I'm so glad you got it sorted out @ScareKrwoe , good night! 👍

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @ScareKrwoe , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
